awx/ui_next/src/components/LaunchPrompt/useSteps.js

```diff
@@ -31,8 +37,10 @@ export default function useSteps(config, resource, i18n) {
     };
   }, {});
   const isReady = !steps.some(s => !s.isReady);
-  const stepWithError = steps.find(s => s.error);
-  const contentError = stepWithError ? stepWithError.error : null;
+
+  const stepWithError = steps.filter(s => s.contentError);
+
+  const contentError = stepWithError ? stepWithError.contentError : null;
```

Comment:
stepWithError is an array (or atleast looks to be one base on your filter statement above)...it's not an object.  I don't know exactly what you're trying to do, but this won't work.  the line 34 you deleted uses "find" which is like filter, but returns the first item of the array, rather than the array itself.  maybe you meant to use find instead of filter?

Reply:
I am looking into this. But as far as I can tell, I just renamed `error` to `contentError` and adjust those variables.
Currently in the code basis.
```js
  const stepWithError = steps.find(s => s.error);
  const contentError = stepWithError ? stepWithError.error : null;
```
But let me take a better look.